src/fs/file.rs

```diff
@@ -334,3 +335,9 @@ impl AsRawFd for File {
         self.fd.raw_fd()
     }
 }
+
+impl fmt::Debug for File {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "File {{ fd: {fd} }}", fd = self.fd.raw_fd())
+    }
+}
```

Comment:
Please use [`debug_struct`][1] so that we properly handle it when the user pretty-prints the debug output.

[1]: https://doc.rust-lang.org/std/fmt/struct.Formatter.html#method.debug_struct